doc/en/introduction.rst

```diff
@@ -350,7 +350,14 @@ Command line
                                 ......
                             }
       --skip-remaining {cases-on-failed,cases-on-error,suites-on-failed,suites-on-error,tests-on-failed,tests-on-error}
-                            Skip the remaining Testcases/Testsuites/Multitests being executed when a Testcase has failed or raised exception.
+                            Make Testplan skip certain testcases from execution upon exception ("error") or assertion failed ("failed"). Here "failed" includes "error" situation.
```

Comment:
This option makes testplan to break from the current execution flow on error/failed and skip remaining iterations at certain level.

When on-error is specified, skip-remaining is triggered when exception("error") happens during testing. When on-failed is specified, skip-remaining is triggered upon both exception("error") and test failure("failed").

To --skip-remaining cases, means skip remaining testcases in the same testsuite, and resume with next testsuite.

Reply:
To --skip-remaining suites, means skip remaining testcases in the current testsuite, and remaining testsuites in the current Multitest, and resume with next Multitest.

To --skip-remaining tests, means stop everything - remaining testcases/testsuites in all the Multitests that are currently executing, and remaining tests that has not yet start executing.